homeassistant/components/upnp/sensor.py

```diff
@@ -18,7 +18,10 @@
     LOGGER,
     PACKETS_RECEIVED,
     PACKETS_SENT,
+    ROUTER_IP,
+    ROUTER_UPTIME,
     TIMESTAMP,
+    WAN_STATUS,
 )
 
 SENSOR_TYPES = {
```

Comment:
Next improvement; use `SensorEntityDescription` for this

Reply:
Assuming you mean not for this PR. :-)

Can you provide some more info on what you mean? Did Google search on it but did not seem to find much info on this.

Reply:
I guess you can do it here as well :smile: 
Do a codebase search instead, over the past few weeks a lot of integrations have moved to that.
It will really simplify the descriptions of the entities that is being done (including the disabled by default setting)

Reply:
LOL.

Ok, just pushed a commit for it. Haven't been able to test local yet but the tests themselves passed. Figured I'll already push it so you can review and let me know what I need to change. :-)